modules/ppcp-api-client/src/Endpoint/class-requesttrait.php

```diff
@@ -36,6 +39,54 @@ private function request( string $url, array $args ) {
 			$args['headers']['PayPal-Partner-Attribution-Id'] = 'Woo_PPCP';
 		}
 
-		return wp_remote_get( $url, $args );
+		$response = wp_remote_get( $url, $args );
+
+		if ( is_wp_error( $response ) ) {
+			$this->logger->error( $response->get_error_code() . ' ' . $response->get_error_message() );
+			return $response;
+		}
+
+		$this->logger->info( $this->request_response_string( $url, $args, $response ) );
```

Comment:
Maybe better to use some lower level like `->debug`, so that user can filter it.

Reply:
Done.